pkg/gadgets/traceloop/gadget.go

```diff
@@ -123,6 +123,11 @@ func (t *Trace) Start(trace *gadgetv1alpha1.Trace) {
 		echo "-:pfree_uts_ns" >> /sys/kernel/debug/tracing/kprobe_events 2>/dev/null || true
 		echo "-:pcap_capable" >> /sys/kernel/debug/tracing/kprobe_events 2>/dev/null || true
 
+		# Increase the opened files limit to avoid getting this error on Mariner:
+		# error while loading "tracepoint/raw_syscalls/sys_enter" (too many open files):
+		# This value is the minimal power of 2 to get rid of above error.
+		ulimit -n 2048
```

Comment:
I would not comment on the OS (Mariner) but on the specific reasons, e.g. traceloop starts 128 traces and the number of fds for each of them is proportional to the CPU count. If that's the case, then instead of using the minimum value that works for you, we should use a bigger value that's guaranteed to work regardless of the amount of CPUs. Or even `unlimited`.

Reply:
Since you use ulimit, maybe update the comment about CAP_SYS_RESOURCE in:
https://github.com/kinvolk/inspektor-gadget/blob/main/cmd/kubectl-gadget/deploy.go#L294-L295

```
              # Needed by setrlimit in gadgettracermanager and by the traceloop gadget.
              - SYS_RESOURCE
```

Reply:
>If that's the case, then instead of using the minimum value that works for you, we should use a bigger value that's guaranteed to work regardless of the amount of CPUs.

Maybe we can be smarter and multiplies the actual value by number of CPU?